contracts/libraries/NFTSVG.sol

```diff
@@ -238,7 +238,7 @@ library NFTSVG {
             curve = curve4;
         } else if (tickRange <= 100) {
             curve = curve5;
-        } else if (tickRange <= 10_000) {
+        } else if (tickRange <= 1_000) {
```

Comment:
i just realized, we should look at the `(tickUpper - tickLower) / tickSpacing`, rather than the absolute difference, because it depends on the tick spacing. 5 ticks difference is not possible in a 200 tick spacing pool

Reply:
Does tick spacing effect the price at each tick (in absolute terms?)

Reply:
also, we may want to use powers of 2 instead of powers of 10, 8 curves with powers of 2 is 256, and 256 tick spacings is == 167x price diff with high fee and 1.29 with low fee